main.tf

```diff
@@ -26,27 +27,26 @@ resource "google_container_node_pool" "this" {
   project    = google_container_cluster.this.project
   cluster    = google_container_cluster.this.name
   location   = google_container_cluster.this.location
-  node_count = var.GKE_NUM_NODES
+  node_count = 3
 
   node_config {
-    machine_type = var.GKE_MACHINE_TYPE
+    machine_type = "n1-standard-2"
```

Comment:
hardcode?

Reply:
agree. 
in this case i must to add in variable.tf

variable "GKE_MACHINE_TYPE" {
  description = "Machine type for GKE nodes"
  default     = "n1-standard-2"
}

but this will be hardcode

Reply:
variable and default value is not hardcode - but right way

Reply:
this is only for test 
i do init than plan, i will check how this will affect the infrastructure and changes, and after that, I will already make an apply.

I basically thought that I need to make some changes to the file